src/style.css

```diff
@@ -0,0 +1,56 @@
+/* Reset 'body' indents */
```

Comment:
I'm spending extra time reading this comment, but if I just look at the code and understand the same thing and faster